tests/unit/test_slide.py

```diff
@@ -175,7 +175,7 @@ def it_has_largeimage_tilesource(self, tmpdir):
             tmpdir, PILIMG.RGBA_COLOR_500X500_155_249_240, use_largeimage=True
         )
 
-        assert slide._tile_source.name == "pilfile"
+        assert slide._tile_source.name == "pil"
```

Comment:
legit change due to this commit 
https://github.com/girder/large_image/commit/8f7e889e56cba6f6339d4e85659921e0626612c2